spec/src/main/asciidoc/javase/javase.asciidoc

```diff
@@ -2,7 +2,7 @@
 --
 This part of the document specifies additional rules and features when using CDI in Java SE.
 All content defined in <<part_1>> applies to this part.
-// TODO are implementations required to support Full?
+Therefore, any implementation of CDI in Java SE has to be built on top of {cdi_full}.
```

Comment:
Well implementations of CDI Lite are also in Java SE :-) The name of this chapter is hard to use frankly.

I'd say something like:

```suggestion
In particular, implementations must provide support for the entire {cdi_full} specification.
Support for CDI in Java SE that only requires {cdi_lite} may be added in subsequent versions of CDI.
```

2nd line being optional of course :-)